ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -213,7 +271,12 @@ open class BottomCommandingController: UIViewController {
     public override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
         super.viewWillTransition(to: size, with: coordinator)
         if presentedViewController != nil {
-            dismiss(animated: false)
+            dismiss(animated: false) { [weak self] in
+                guard let strongSelf = self else {
+                    return
+                }
```

Comment:
minor: just use a ? on the next line instead of this to simplify?

Reply:
We need to pass in a non-optional self to the delegate. That's why I'm getting it here.